### ISSUE TITLE
Update: B-21 weapon

### ISSUE THREAD

**Issue by HunterDNiu** [PygmalionOfCyprus/cmo-db-requests#4278]
### DB Selector

DB3K

### Affected DBID(s)

Aircraft 6748

### Summary of Changes

Change loadouts to combinations of the options below:

- For the secondary weapon bays: 

12x AIM-120D3,
12x AIM-260, or
6x AIM-120D3 or AIM-260 and 6x AGM-88E/G or SiAW(mixed).

- For the primary weapon bay:

8x AGM-158A/B/B2/C,
8x AGM-181A,
8x HAWC/HAWC AShM,
16x AIM-120D3,
16x AIM-260,
16x B61-12/13(#4279),
16x B83,
8x AGM-154A/C/C1, 
16x AGM-88E/G or SiAW,
120x SDB/SDB laser/SDB II,
16x CBU-103/105,
4x GBU-18B/C,
1x MOP,
12x JDAM DEMP/OEMP,
8x JDAM Mk84/BLU-109B,
8x LJDAM Mk84,
40x JDAM Mk82,
40x LJDAM Mk82,
8x QUICKSINK, or
8x Quickstrike-ER/Quickstrike-P.

### Sources

https://www.twz.com/b-21-raider-has-flown-for-the-first-time
https://s3.documentcloud.org/documents/24400278/2023annual-report.pdf

**Comment by caelunshun** [comment]
I agree with the loadout reductions; the capacity should generally be half that of the B-2.

What's the source for the secondary weapons bay you mention? I couldn't find anything in the article or the annual report.
